feinstaub/sensors/models.py

```diff
@@ -125,6 +125,14 @@ def __str__(self):
     ('noise_L01', 'Sound level L01'),
     ('noise_L95', 'Sound level L95'),
     ('noise_Leq', 'Sound level Leq'),
+    ##gas sensor
+    ('co_kohm', 'CO in kOhm'),
+    ('co_ppb','CO in ppb'),
+    ('co2','eCO2 in ppm'),
+    ('no2_kohm', 'NO2 in kOhm'),
+    ('no2_ppb','NO2 in ppb'),
+    ('ozone','O3 in ppb'),
+    ('so2_ppb','SO2 in ppb'),
```

Comment:
Just to make sure there is no more rounds trip:

1. Are ozone and s02 never measured in other units?
2. And co2 is also always in parts per million?

Reply:
@kilemensi For now, that's what we know from existing networks...
 
 Ozone & SO2 appear only on AIRNOW and CO2 only on smart citizen.  The rest with multiple unit appear in both

Reply:
So if a new networks states they record SO2 using g/m3..we shall add it.  We can't know all

Reply:
Then for ozone at least  lets name it `ozone_ppb` I think @KhadijaMahanga